client/landing/subscriptions/components/comment-list/comment-row.tsx

```diff
@@ -1,47 +1,55 @@
 import { Gridicon } from '@automattic/components';
-import { useMemo } from 'react';
+import { memo, useMemo } from '@wordpress/element';
```

Comment:
Is there a reason to import react stuff from `@wordpress/element`?

Reply:
I was under the impression that it's considered[ best practice](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-element/), to use the abstraction layer. If the project ever decides to shift to another framework, such as Preact, it would only mean doing the update in `@wordpress/element`. But searching the code, I see that it would be a lot more work because `react` and `@wordpress/element` seem to be used interchangeably 😀

Reply:
I import them from react now in the new implementation. Saves a line :)